clusterloader2/pkg/prometheus/manifests/kubemark/kubemark-master-service.yaml

```diff
@@ -14,6 +14,8 @@ spec:
   ports:
     - name: apiserver
       port: 443
+    - name: etcd
+      port: 2379
```

Comment:
[pull-kubernetes-kubemark-e2e-gce-big](https://testgrid.k8s.io/presubmits-kubernetes-blocking#pull-kubernetes-kubemark-e2e-gce-big) is failing and the starting time seems to align with this PR

this change seems to conflict with the latter duplicate `etcd` port:

from https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-kubemark-e2e-gce-big/1133114154450685954
```
W0527 21:23:36.157] F0527 21:23:36.156399   85477 clusterloader.go:233] Error while setting up prometheus stack: error while applying (/go/src/k8s.io/perf-tests/clusterloader2/pkg/prometheus/manifests/kubemark/kubemark-master-service.yaml): Service "kubemark-master" is invalid: [spec.ports[5].name: Duplicate value: "etcd", spec.ports[5]: Duplicate value: core.ServicePort{Name:"", Protocol:"TCP", Port:2379, TargetPort:intstr.IntOrString{Type:0, IntVal:0, StrVal:""}, NodePort:0}]
```